CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ enable_language( CXX )
 include( CheckCXXSourceCompiles )
 include( CTest )
 
+find_package(pybind11 REQUIRED)
```

Comment:
Use `git submodule` to add linking to pybind11

Reply:
Personally I would prefer to assume that `pybind11` is installed on the system in question, and just use the `find_package(pybind11 REQUIRED)`. I agree we can use `git submodule` for `pybind11` - but that would be a "general decision" - and not just to support Travis building.